src-java/server42/server42-control-messaging/src/main/java/org/openkilda/server42/control/messaging/flowrtt/PushSettings.java

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.server42.control.messaging.flowrtt;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+
+
+@Value
+@Builder
+@NoArgsConstructor(force = true, access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+public class PushSettings extends Message {
+    @JsonProperty("packet_generation_interval_in_ms")
+    Integer packetGenerationIntervalInMs;
```

Comment:
why not primitive here?

Reply:
Because I don't want to send all list of settings every time. In that case, I just can use null and forget about setting param if I need to.